examples/todo.yaml

```
@@ -0,0 +1,67 @@
+  from rich import print
+
+commands:
+  create:
+    help: Create a new database with tasks table
+    params:
+      - name: str = typer.Option(..., prompt=True, confirmation_prompt=True)
+    run: |
+      db_path = Path(f"{name}.db")
+      conn = sqlite3.connect(db_path)
+      conn.execute("CREATE TABLE tasks (id INTEGER PRIMARY KEY, task TEXT NOT NULL, done BOOLEAN NOT NULL)")
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)